src/sync/SSEHandler/NotificationKeeper.js

```diff
@@ -0,0 +1,56 @@
+import { PUSH_CONNECT, PUSH_DISCONNECT, PUSH_DISABLED, ControlTypes } from '../constants';
+
+const controlPriMatcher = /control_pri$/;
```

Comment:
Similar to the one in SSE client. Also we should have other regexes so maybe follow that name convention? 

Given it's constant, should be all capitalized letters.

Reply:
changed these new regex to follow the naming convention: X_CHANNEL_REGEX